workflows/sample/tasks/hifiasm_trio.wdl

```diff
@@ -45,11 +45,10 @@ task hifiasm_trio_assemble {
     File hap2_p_ctg        = "~{prefix}.dip.hap2.p_ctg.gfa"
     File hap2_p_ctg_lowQ   = "~{prefix}.dip.hap2.p_ctg.lowQ.bed"
     File hap2_p_noseq      = "~{prefix}.dip.hap2.p_ctg.noseq.gfa"
-    File p_utg             = "~{prefix}.dip.p_utg.gfa"
-    File r_utg             = "~{prefix}.dip.r_utg.gfa"
-    File ec_bin            = "~{prefix}.ec.bin"
-    File ovlp_rev_bin      = "~{prefix}.ovlp.reverse.bin"
-    File ovlp_src_bin      = "~{prefix}.ovlp.source.bin"
+    File p_utg_noseq       = "~{prefix}.dip.p_utg.noseq.gfa"
+    File p_utg_lowQ        = "~{prefix}.dip.p_utg.lowQ.bed"
+    File r_utg_noseq       = "~{prefix}.dip.r_utg.noseq.gfa"
+    File r_utg_lowQ        = "~{prefix}.dip.r_utg.lowQ.bed"
```

Comment:
Added the `noseq` versions of the unitig graphs and the corresponding low quality region bed files.